tests/unittests/cmd/test_main.py

```diff
@@ -267,12 +267,20 @@ def test_main_sys_argv(
             # Non-cloud-config
             (mock.Mock(), "#!/bin/bash\n  - echo hello", True),
             # Something that won't decode to utf-8
-            (mock.Mock(), os.urandom(100), True),
+            (mock.Mock(), "RANDOM100", True),
```

Comment:
This an the next don't match the comment "won't decode to utf-8".

Reply:
Push an update